app/services/random_token_generator.rb

```diff
@@ -0,0 +1,14 @@
+class RandomTokenGenerator
+  def initialize(model_name, model_token)
+    @model_name = model_name.constantize
+    @model_token = model_token
```

Comment:
rename this to token_name
as one model can have 2 token like user.

Reply:
Done